biolink-model.yaml

```diff
@@ -1374,6 +1374,14 @@ slots:
       - SEMMEDDB:LOCATION_OF
       - WIKIDATA:276
 
+  has sequence location:
+    is_a: related to
+    description: >-
+      holds between two genomic entities when the subject can be localized in sequence coordinates on the object. For example, between an exon and a chromosome.
```

Comment:
where do less formalized objects such as contigs stand?

Reply:
Also noting that avoiding overloading this to mean "this is necessarily the actual sequence"  could help.